burn-import/tests/data/concat/concat.rs

```diff
@@ -0,0 +1,31 @@
+// Generated by integration tests
+use burn::{
+    module::Module,
+    tensor::{backend::Backend, Tensor},
+};
```

Comment:
I don't think we should commit this test, we already have unit tests that check the codegen part. But I agree that we are missing a step to correctly test the onnx parsing part, which is hard to validate without a test like this.

Reply:
True. I am open for suggestions to improve. I agree it might be redundant and it 

We could commit ONNX files without verifying the output. Although it's a true guarantee that the onnx file is properly converted. The unit tests that you're mentioning check if the node is converted to source code directly. However, onnx-to-source is an integration test, good for the end to end. 

We can take this up in a separate PR or discussion.